src/components/organisms/cards/cards.html

```diff
@@ -0,0 +1,58 @@
+<div class="container">
+  <div class="container cards">
```

Comment:
```suggestion
  <div class="cards">
```
2つ `container` が存在する意味は薄いので、こちらは消しても大丈夫です。